web-react-ts/src/pages/camps/AssignAdmin.tsx

```diff
@@ -12,27 +9,20 @@ const AssignAdmin = () => {
   const onOpenFirstModal = () => setIsOpenFirstModal(true)
   const onCloseFirstModal = () => setIsOpenFirstModal(false)
 
-  const firestore = useFirestore()
-  const userCollection = collection(firestore, 'users')
-  const { data: users } = useFirestoreCollectionData(userCollection, {
-    idField: 'id',
-  })
+  const loading = false
 
-  const loading = !users
-
-  const usersData: UserData[] = []
-
-  users?.forEach((user: DocumentData) => {
-    usersData.push({ id: user.id, ...user.data() })
-  })
+  const data = [
+    { id: '1', name: 'John Doe' },
+    { id: '2', name: 'Jane Doe' },
+  ]
```

Comment:
why dont you rather have a separate file to hold this JSON file , fetch from that file and maintain the previous code